clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -181,6 +181,14 @@ New checks
   points in a coroutine. Such hostile types include scoped-lockable types and
   types belonging to a configurable denylist.
 
+- New :doc:`modernize-string-find-startswith
+  <clang-tidy/checks/modernize/string-find-startswith>` check.
+
+  Checks whether a ``std::string::find()`` or ``std::string::rfind()`` (and
+  corresponding ``std::string_view`` methods) result is compared with 0, and
+  suggests replacing with ``starts_with()``. This is both a readability and a
```

Comment:
"This is both a readability and a performance issue." no need for that in release notes